kmip/services/server/session.py

```diff
@@ -51,6 +51,9 @@ def __init__(self, engine, connection, name=None):
             kwargs={}
         )
 
+        if name is None:
+            name = threading.current_thread().name
```

Comment:
Ok, I see, thanks.

By the way:
- does there any mailing list to ask the newbie questions? Like these ones:
- newbie questions:
  - what are/will you use for the threads? I'm trying to use the  SocketServer/AsynchronousMixins where KMIP session is run by it's RequestHandler. In this mode SocketServer makes thread per connection (with a distict thread name, but for a while I have not found the way to format this name). 
 - you've told _parent thread, MainThread_ -- does it mean that running server has to be separated  from the launching process into a dedicated thread? Can it run directly in the launching process?